src/pages/CreateBurnerWallet.tsx

```diff
@@ -52,6 +53,13 @@ const buttonWithStatus = classnames(
 )
 const buttonClass = classnames(display('block'), width('w-full', 'sm:w-64'))
 
+const getAxiosErrorMessage = (error: unknown) => {
+  return (
+    (error as AxiosError<{ message: string }>).response?.data.message ||
+    'Verification failed'
```

Comment:
I'd just return the `error` unchanged if it's not an `AxiosError`. Also, can you typecheck for `AxiosError` instead of casting? And remember, default errors are bad

Reply:
Left only one default message that will appear under the input. Otherwise, all errors will be handled by `handleError` and toastified.